.storybook/components/DataTable/DataTableData.jsx

```diff
@@ -1,15 +1,15 @@
 import { STATUS_OPTIONS } from '@airbnb/lunar/src/components/DataTable/constants';
 
-export function generateRandomData() {
-  return new Array(50).fill(0).map((x, i) => ({
+export function generateRandomData(n = 800) {
```

Comment:
Return bigger (and more random) data so we can test performance on large dataset.